ReadPDFFileV2 unescaping (#33258)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_14_20.md

#### Scripts

##### ReadPDFFileV2
- Updated the Docker image to: *demisto/readpdf:1.0.0.89247*.

> - Added the argument **unescape**. When set to *true*, the script will unescape the URLs that have been escaped (Ignoring them). Default is 'True'. 

Reviewer: ```suggestion
- Added the argument **unescape**. When set, the script will unescape the URLs that have been escaped (by ignoring them). Default is 'true'. 
```
